lib/response.js

```diff
@@ -61,18 +61,41 @@ res.status = function(code){
  *      last: 'http://api.example.com/users?page=5'
  *    });
  *
- * @param {Object} links
+ *    res.links([{
+ *          href: 'http://api.example.com/users?page=2',
+ *          rel: 'next',
+ *          title: 'next chapter',
+ *          type: 'text/plain;charset=UTF-8'
+ *      }, {
+ *          href: 'http://api.example.com/users?page=5',
+ *          rel: 'last',
+ *          title: 'last chapter',
+ *          type: 'text/plain;charset=UTF-8'
+ *      }]);
+ *
+ *
+ * @param {Object|Array} links
  * @return {ServerResponse}
  * @api public
  */
 
-res.links = function(links){
+res.links = function(links) {
   var link = this.get('Link') || '';
   if (link) link += ', ';
-  return this.set('Link', link + Object.keys(links).map(function(rel){
-    return '<' + links[rel] + '>; rel="' + rel + '"';
-  }).join(', '));
-};
+  if (Array.isArray(links)) { 
+    for (var i = 0; i < links.length; i++) { 
+      if (i !== 0) link += ', '; // do not prepend ', ' if it's the first link
+      link += '<' + links[i].href + '>; ';
+      delete links[i].href; // already appended
+      link += Object.keys(links[i]).map(function(key) { return key + '="' + links[i][key] + '"'; }).join('; ');
```

Comment:
Any preference on how to implement this? Should we have a whitelist of valid parameter for this, as per [RFC 5988](https://tools.ietf.org/html/rfc5988#page-6) or is checking for alpha characters sufficient?